testem.js

```diff
@@ -13,7 +13,6 @@ module.exports = {
         // --no-sandbox is needed when running Chrome inside a container
         process.env.CI ? '--no-sandbox' : null,
         '--headless',
-        '--disable-gpu',
```

Comment:
Was this a blocker at some point?

Reply:
My tests were blowing up when running `ember t` locally. Removing it makes them work. 

Unsure what to do with this commit as of right now.

Reply:
Have removed this commit as it doesn't seem to be blowing up now. No idea what's going on there.